helpers/str/str.go

```diff
@@ -0,0 +1,13 @@
+package str
```

Comment:
Instead of `str` `text` would be more descriptive 💭

Reply:
According to this [comment](https://github.com/gutakk/go-google-scraper/pull/25#discussion_r536014123). I removed `str package`